modules/formulize/include/js/conditional.js

```diff
@@ -114,7 +114,7 @@ function checkCondition(relevantElementSet, elementValuesForURL) {
         }
         elementIdsSep = ',';
     }
-	return jQuery.post(FORMULIZE.XOOPS_URL+"/modules/formulize/formulize_xhr_responder.php?uid="+FORMULIZE.XOOPS_UID+"&op=get_element_row_html&elementId="+elementIds+"&entryId="+entryId+"&fid="+fid+elementValuesForURL);
+	return jQuery.post(FORMULIZE.XOOPS_URL+"/modules/formulize/formulize_xhr_responder.php?uid="+FORMULIZE.XOOPS_UID+"&sid="+FORMULIZE.SCREEN_ID+"&op=get_element_row_html&elementId="+elementIds+"&entryId="+entryId+"&fid="+fid+elementValuesForURL);
```

Comment:
The critical passing of screen id to the conditional rendering request. Note that there is security about whether the user has permission to view the element, otherwise we don't render anything. No security specifically on the validity of the screen id, however screens don't contain any data related to permission-controlled activities, and all such activities (like writing data, etc) validate the user prior to doing the action. So I think we're in the clear for accepting the screen id as is.